### DISCUSSION TITLE
Riverside data management EXPLOIT

### DISCUSSION THREAD

**Discussion by U-Nazu** [3kh0/ext-remover#1014] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
So if your school uses this riverside data is a kiosk app
1. Open app
2. Go to tests and on login type a URL
3. Double-click the link and it should say "go to URL"
4. Now hover over it and press enter and TAP THE TOP-LEFT of the chromebook somewhat frimly do step four over if it fails
If you need help please comment I will try my best

**Comment by Blobby-Boi** [comment]
I double click it nothing pops up
theres like 10 textboxes in the login page wchich one am i supposed to use

**Reply by U-Nazu** [reply]
Well when you get to the prompt it should be the very first name and you should type in a URL 
For at least my school we can double click and it will show like the Google double click I'm not sure it will work on every Chromebook like my friend said. Please comment for more questions.

**Reply by Blobby-Boi** [reply]
I double click and nothing pops up it just selects it

**Reply by VFXKLIPS** [reply]
Yo how do you do it can you please help me, tell me step by step

**Reply by jazzymakesstuff** [reply]
www link and then hit space once

**Comment by YoiseAndZane** [comment]
actually, its to get your thumb on the top left of the screen and swipe it up and down on the screen, sometimes this wont work because of screen size so just do crtl and + to increase the screen and it might fix it

**Reply by jazzymakesstuff** [reply]
video please
